lib/read/lib/Cardano/Wallet/Read/Block/Gen/Build.hs

```diff
@@ -0,0 +1,432 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+
+module Cardano.Wallet.Read.Block.Gen.Build
+    ( ChainF
+    , TxBuild (..)
+    , ChainBuild (..)
```

Comment:
```suggestion
    , TxF
```

In order to maintain the abstraction boundary, only the types `TxF` and `ChainF` should be exported — the `TxBuild` and `ChainBuild` types are internal to this module.